recipe/linter_hints/hints.toml

```diff
@@ -91,9 +91,133 @@ astropy = """\
+    Please use `xorg-xorgproto` instead. It contains the latest versions of all the proto \
+    packages and bundles them all together."""
+xorg_inputproto = """\
+    The use of `xorg-inputproto` is deprecated. \
+    Please use `xorg-xorgproto` instead. It contains the latest versions of all the proto \
+    packages and bundles them all together."""
+xorg_kbproto = """\
+    The use of `xorg-kbproto` is deprecated. \
+    Please use `xorg-xorgproto` instead. It contains the latest versions of all the proto \
+    packages and bundles them all together."""
+xorg_presentproto = """\
+    The use of `xorg-presentproto` is deprecated. \
+    Please use `xorg-xorgproto` instead. It contains the latest versions of all the proto \
+    packages and bundles them all together."""
+xorg_printproto = """\
+    The use of `xorg-printproto` is deprecated. \
+    Please use `xorg-xorgproto` instead. It contains the latest versions of all the proto \
+    packages and bundles them all together."""
+xorg_randrproto = """\
+    The use of `xorg-randrproto` is deprecated. \
+    Please use `xorg-xorgproto` instead. It contains the latest versions of all the proto \
+    packages and bundles them all together."""
+xorg_recordproto = """\
+    The use of `xorg-recordproto` is deprecated. \
+    Please use `xorg-xorgproto` instead. It contains the latest versions of all the proto \
+    packages and bundles them all together."""
+xorg_renderproto = """\
+    The use of `xorg-renderproto` is deprecated. \
+    Please use `xorg-xorgproto` instead. It contains the latest versions of all the proto \
+    packages and bundles them all together."""
+xorg_resourceproto = """\
+    The use of `xorg-resourceproto` is deprecated. \
+    Please use `xorg-xorgproto` instead. It contains the latest versions of all the proto \
+    packages and bundles them all together."""
+xorg_scrnsaverproto = """\
+    The use of `xorg-scrnsaverproto` is deprecated. \
+    Please use `xorg-xorgproto` instead. It contains the latest versions of all the proto \
+    packages and bundles them all together."""
+xorg_trapproto = """\
+    The use of `xorg-trapproto` is deprecated. \
+    Please use `xorg-xorgproto` instead. It contains the latest versions of all the proto \
+    packages and bundles them all together."""
+xorg_videoproto = """\
+    The use of `xorg-videoproto` is deprecated. \
+    Please use `xorg-xorgproto` instead. It contains the latest versions of all the proto \
+    packages and bundles them all together."""
+xorg_windowswmproto = """\
+    The use of `xorg-windowswmproto` is deprecated. \
+    Please use `xorg-xorgproto` instead. It contains the latest versions of all the proto \
+    packages and bundles them all together."""
+xorg_xcb_proto = """\
+    The use of `xorg-xcb-proto` is deprecated. \
+    Please use `xorg-xorgproto` instead. It contains the latest versions of all the proto \
+    packages and bundles them all together."""
```

Comment:
The XCB protocol files are an exception: they are the only package that has *not* been merged into the generic xorgproto package. So this specific lint should be removed.

Due to the above, this second item is no longer relevant, but also: there is a legacy `xorg-xcb-proto` feedstock, but the proper name for this package is just `xcb-proto`: we had to change the name to maintain compatibility with Anaconda's `defaults`, back in the day. The `xorg-xcb-proto` feedstock should be archived.

Reply:
(E.g., check out: https://www.x.org/releases/individual/proto/?C=M;O=D )

Reply:
Thanks Peter! 🙏 

Have tried to propose some changes based on your feedback. Please let me know whether those need make sense or need changes

Reply:
Yeah, what you propose looks good. *Hopefully* there isn't anything that actually uses the `xorg-xcb-proto` package, since we realized that it needed to be renamed almost right off the bat, but it doesn't hurt to add a lint for it.

Reply:
Great, have gone ahead and committed it. Please let me know if there is anything else